src/logger.ts

```diff
@@ -126,14 +126,15 @@ export const Print = {
     sms?: boolean,
     meta?: string | null
   ): string {
+    const associateLink = link.associateLink ? link.associateLink : '';
```

Comment:
since associateLink is not mandatory, define it here as empty string incase its undefined to avoid the linting error